src/app/routes/regex/index.js

```diff
@@ -11,13 +11,20 @@ const assetUriRegex = '[a-z-_]{0,}[0-9]{8,}';
 
 const variantRegex = '/simp|/trad|/cyr|/lat';
 
-export const articleRegexPath = `/:service(${serviceRegex})/articles/:id(${idRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+const articleLocalMapping = {
+  cymrufyw: 'erthyglau',
+  naidheachdan: 'sgeulachdan',
+};
+
+const articleLocalRegex = `articles|${articleLocalMapping.cymrufyw}|${articleLocalMapping.naidheachdan}`;
```

Comment:
We could simplify this to:

```suggestion
const articleLocalRegex = `articles|erthyglau|sgeulachdan`;
```

Reply:
Ok. Was thinking using the map makes it clearer what translation of "articles" we are using.